no-computer.md

```diff
@@ -58,12 +91,16 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+There's a lot of room for this DSL to grow since someone can always create
+new designs of Lego pieces. 
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+Not applicable?
```

Comment:
In this case, I would agree that it is not applicable; however, I took the assignment to mean that we should create or extend a physical DSL making these questions applicable. Out of curiosity, if you were to re-invent or change Legos, what would you do to make it more expressive and powerful?